### ISSUE TITLE
Express error handling does not work properly

### ISSUE THREAD

**Issue by mStirner** [OpenHausIO/backend#26]
https://github.com/OpenHausIO/backend/blob/dev/routes/index.js#L119:

```
TypeError: res.status is not a function
    at /opt/OpenHaus/backend/routes/index.js:119:17
    at Layer.handle [as handle_request] (/opt/OpenHaus/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/OpenHaus/backend/node_modules/express/lib/router/index.js:317:13)
    at /opt/OpenHaus/backend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/OpenHaus/backend/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/OpenHaus/backend/node_modules/express/lib/router/index.js:275:10)
    at expressInit (/opt/OpenHaus/backend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/opt/OpenHaus/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/OpenHaus/backend/node_modules/express/lib/router/index.js:317:13)
    at /opt/OpenHaus/backend/node_modules/express/lib/router/index.js:284:7
```
But should, example from express documentation: https://expressjs.com/de/guide/error-handling.html

**Comment by mStirner** [comment]
"Middleware" changed for error logging:

```js
        app.use((error, req, res) => {

            console.error(error.stack);

            res.status(500);

            if (process.env.NODE_ENV !== "production") {
                res.end(err.message);
            } else {
                res.end();
            }

        });
```

No bug found in code above, observe where the error message `res.status ...` come from (component, auth, etc...)

**Comment by mStirner** [comment]
Fixed in #54
